codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -2231,6 +2231,7 @@ data Wrapped a = Wrapped
     { state :: a
     , playbackSpeed :: Double
     , mouseMovedTime :: Double
```

Comment:
Consider renaming `mouseMovedTime` to something like `lastInteractionTime`?  It's now tracking time since any user event, not just mouse movement.